src/os/bluestore/BlueStore.cc

```diff
@@ -6783,19 +6767,20 @@ void BlueStore::_main_bdev_label_remove(Allocator* an_alloc)
 }
 
 int BlueStore::_check_or_set_bdev_label(
```

Comment:
All calls to _check_or_set_bdev_label are wrapped with if (bdev_support_label()) verification - IMO we can move it inside this func and kill the relevant wrapper on top of BlockDevice at BlueFS class.